runtime/interpreter/decode.go

```diff
@@ -411,8 +411,9 @@ func (d *Decoder) decodeDictionary(v interface{}, path []string) (*DictionaryVal
 
 	entries := NewStringValueOrderedMap()
 
-	var deferred *orderedmap.StringStringOrderedMap
+	var deferred *orderedmap.StringStructOrderedMap
```

Comment:
curious to know, whether using a string-interface map with nil values be more efficient, because we are not writing the values anyway?

Reply:
Great concern, @dete had asked the same.

In Go, an `interface{}` is 8/16 bytes, whereas an `struct{}` is literally zero size, see e.g. https://stackoverflow.com/questions/22770114/any-difference-in-using-an-empty-interface-or-an-empty-struct-as-a-maps-value